lib/api/charts.php

```diff
@@ -162,9 +162,9 @@ function if_chart_exists($id, $callback) {
         }
         $data = $app->request()->getBody();
         try {
-            $filename = $chart->writeData($data);
+            $chart->writeData($data);
             $chart->save();
-            ok($filename);
+            ok();
```

Comment:
this changes the existing API signature, but I'm not sure we ever used the returned filename for anything...

Reply:
yes, this was intentional